package-lock.json

```diff
@@ -3590,6 +3590,11 @@
         "through": "~2.3.6"
       },
       "dependencies": {
+        "diff": {
+          "version": "1.0.8",
```

Comment:
No where in package.json to change this version. Going to see if github complains later after merge.